.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: verilator
+  name: verilator-lint
+  description: Runs verilator Docker image to lint (System) Verilog designs
+  args: [--lint-only, -Wall]
+  language: docker_image
+  entry: verilator/verilator:v5.026
+  types_or: ["verilog", "system-verilog"]
```

Comment:
For context, see https://github.com/pre-commit/identify/blob/52ba50e2a234147d85320b6e1cff065b30377020/identify/extensions.py

Reply:
Maybe you should add 'vlt' as a verilator control file to that master file, then here?

Reply:
Is that file format only used for verilator? Does it contain verilog code?

Reply:
.vlt is only for Verilator commands, not verilog.

Reply:
Gotcha. Why does that need to be passed to verilator? Is it a config file that we should pass as specific argument?

Reply:
A .vlt file is optional, I only mentioned it because this file seems to mention different file extension types.

Reply:
Ok. This filter is only for files to pass for analysis, so I don't think we need to mention .vlt here. If users want to use such a file, they pass it via [args](https://pre-commit.com/#hooks-args).